CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+CHANGES IN VERSION 1.10.0 (from 1.9.0)
+===================================
+
+Appium 1.10.0 is a minor release.
+
+#### General
+* Bring Espresso Driver out of beta and into general availability
+* Discontinue support for Xcode 8
+
+#### iOS
+* Add biometrics (touchId, faceId) [methods](http://appium.io/docs/en/commands/mobile-command/#ios-xcuitest-only) that can enroll/unenroll biometrics and send matching/non-matching biometric inputs
+
+#### Android
+* Add [mobile:](http://appium.io/docs/en/commands/mobile-command) command for performing editor actions [#428](https://github.com/appium/appium-android-driver/pull/428)
+* Send custom keystore [capabilities] to ADB (useKeystore, keystorePath, keystorePassword, keyAlias, keyPassword) [#431](https://github.com/appium/appium-android-driver/pull/431)
```

Comment:
This was actually a regression issue

Reply:
Did it fix something since last release, or was the fix for a bug that occurred between releases and I can just take it out.

Reply:
it's ok to take it out